x-pack/plugins/case/server/services/index.ts

```diff
@@ -149,14 +149,8 @@ export class CaseService {
       }
     },
     getUser: async ({ request, response }: GetUserArgs) => {
-      let user;
-      try {
-        this.log.debug(`Attempting to authenticate a user`);
-        user = await authentication!.getCurrentUser(request);
-      } catch (error) {
-        this.log.debug(`Error on GET user: ${error}`);
-        throw error;
-      }
+      this.log.debug(`Attempting to authenticate a user`);
+      const user = authentication!.getCurrentUser(request);
```

Comment:
__note:__ `getCurrentUser` is synchronous and doesn't throw errors anymore, but it can return `null`.